fvtr/fvtr.sh

```diff
@@ -167,6 +167,11 @@ then
 	export AT_WD=$AT_WD
 fi
 
+if [ ! -e "${AT_BASE}" ]
+then
+	export AT_BASE=$(realpath "$(dirname $0)/..")
+fi
```

Comment:
Is it possible that `AT_BASE` dir exists but is still not valid for this script? Perhaps we could drop the if and always set it here instead of in the Makefile as done in e1a7ebe107b0989db.

Reply:
If `AT_BASE` dir exists and is not valid, then I think the tests will fail.
I agree to make a change in the Makefile.